tools/object_generator.go

```diff
@@ -68,6 +71,10 @@ func (gen *ObjectGenerator) run() {
 	}
 
 	for _, s := range sources {
+		if gen.mode == "object-generator" && s == gen.outfile {
+			continue
+		}
+
```

Comment:
interesting .. so we skip parsing the output file, but only for our runtime mode? In which corner case did you run, is this really specific to this mode or should this be done for all modes?

Reply:
Without this check, all objects would be marked as already identified in the next run.

Reply:
ah this makes sense, of course - thanks for clarifying :)

Still, I think we should exclude the output file from parsing for all modes, or do you see any case where we want to include things we generated in a previous run?